.github/workflows/release.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     if: github.repository_owner == 'apache'
     strategy:
       matrix:
-        flink: [1.16.1, 1.17.0]
+        flink: [1.17.0]
```

Comment:
I actually think you need to add `.github/workflows/push_pr.yml` to set the Flink version either to a matrix (with Flink 1.17.1 and potentially Flink 1.18-SNAPSHOT), or at least that version should be updated to 1.17.1 :)

Reply:
Moved it back to 1.17.0 ...

Reply:
> I'll fix this afterwards :)

I'm an idiot and got confused by Flink 1.16.1, there's no Flink 1.17.1 yet 😅

Reply:
No worries at all